ClassMoodApp/static/js/controllers.js

```diff
@@ -1,106 +1,46 @@
 'use strict';
 var app = angular.module('ClassMoodApp');
 /* Controllers for Angular */
-app.controller('professorClassListController', function($scope, $http, $log) {
-	$http.get('/getProfessorClassList')
-	.then(function successfulCallback(response) {
-		$scope.classList = response.data.results;
-	}, function errorCallBack(response) {
-		$scope.classList = [];
-	});
-});
 
-app.controller('dialogProfessorController', function($scope, $log, $http) {
-	$scope.openDialog = function() {
-		document.getElementById('createclassname').parentNode.MaterialTextfield.change('');
-		$scope.showError = false;
-		var d = document.querySelector('dialog');
-		dialog.showModal();
-	};
+app.controller('liveViewController', function($scope, $log, $http, $timeout) {
```

Comment:
I think we should go the .ts approach for consistency since we've decided to use it going forward.  Even Zach used it in his pull request that's currently out.